### ISSUE TITLE
A correct way to `.ts` content in shaka-player?

### ISSUE THREAD

**Issue by hiren3897** [google/shaka-player#3607]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
yes
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
yes #2683 #3407

**What version of Shaka Player are you using?**
Latest v3.2

**Please ask your question**
We tried to play .ts content in the player but we got an error of  4032

I have already learned the following issues mentioned below:

https://github.com/google/shaka-player/issues/3407
https://github.com/google/shaka-player/pull/2683
https://github.com/google/shaka-player/commit/02a0fbfe109d523e32e2376cc0e1eadd88e55914

I followed the same issues already create but I have confusion on it can you please correct me if I am wrong.

Every time if we want to play .ts or MPEG-2 TS contents we need to load the mux.js file before shaka-player is loaded in the following HTML file?

is mux.js already included in the shaka-player? 

thank you for your answers

**Comment by gregmsanderson** [comment]
Yes, that's right. You need to include mux.js separately since most browsers won't natively play HLS using .ts.

Include that script _before_ Shaka player (though seeing this https://github.com/google/shaka-player/commit/02a0fbfe109d523e32e2376cc0e1eadd88e55914) the order _shouldn't_ matter. But I've continued to load mux.js first, given you previously had to).

You can get mux.js from a CDN like https://cdn.jsdelivr.net/npm/mux.js@5.13.0/dist/mux.min.js
